### DISCUSSION TITLE
Overlay Alerts All/None checkbox

### DISCUSSION THREAD

**Discussion by princealexi25** [smasherprog/EqTool#153] [Ideas]
On top of the checkboxes, it would be really nice to have an All/None checkbox.
Click the checkbox to select all the checkboxes below, uncheck to clear.  For all visual alerts and all audio alerts.

**Comment by smasherprog** [comment]
I agree with this. I am going to work on a redo of how the entire trigger section works.
It will be more like how gina's view is. 
I will also be gina-like features to those as well.

**Comment by smasherprog** [comment]
This is added now closing 
For future, discord is a good place to post ideas too !
https://discord.gg/Z6RPUMNcY2
